frontend/src/metabase/parameters/utils/ui.ts

```diff
@@ -32,7 +32,7 @@ export function buildHiddenParametersSlugSet(
 
 export function getVisibleParameters(
   parameters: UiParameter[],
-  hiddenParameterSlugs?: string,
+  hiddenParameterSlugs?: string | null,
```

Comment:
This makes it accept both `undefined` and `null`. You can also do `hiddenParameterSlugs: string | null = null` if you only need `null`.

Reply:
Much better 😄